### PR TITLE
[fix] clarify use of DB_SOCKET vs. DB_HOST

### LINKED ISSUE #1144: Issue with tutorials/run-laravel-on-appengine-standard/index.md
Issue Description
Setting `DB_SOCKET` in `app.yaml` doesn't work when using PostgreSQL on Cloud SQL.
Laravel uses the `DB_SOCKET` env variable only for MySQL databases (vendor/laravel/lumen-framework/config/database.php).  I used `DB_HOST` instead.

I found the answer here: https://stackoverflow.com/questions/54501305/connecting-laravel-to-gae-cloud-sql-database-with-postgresql

### PR DESCRIPTION
**Pull request by tjwebb**
resolves #1144

### CODE REVIEW

**Comment by ToddKopriva** [comment]
@bshaffer , could you have a look at this suggested change to a document that you worked on?

File: tutorials/run-laravel-on-appengine-standard/index.md [hunk 1]
```diff
@@ -165,7 +165,10 @@ Laravel, you need to manually add the `DB_SOCKET` value to
           DB_DATABASE: YOUR_DB_DATABASE
           DB_USERNAME: YOUR_DB_USERNAME
           DB_PASSWORD: YOUR_DB_PASSWORD
+          ## for MYSQL, use DB_SOCKET:
           DB_SOCKET: "/cloudsql/YOUR_CONNECTION_NAME"
+          ## for PostgreSQL, use DB_HOST:
+          DB_HOST: "/cloudsql/YOUR_CONNECTION_NAME"
```
Comment (bshaffer) [inline tutorials/run-laravel-on-appengine-standard/index.md]: Leave this one commented out so the default is mysql

**Comment by ToddKopriva** [comment]
Thanks for the review, @bshaffer .

File: tutorials/run-laravel-on-appengine-standard/index.md (hunk 1 above)
Comment (tjwebb) [reply tutorials/run-laravel-on-appengine-standard/index.md]: makes sense to me, thanks for taking a look

Comment (ToddKopriva) [reply tutorials/run-laravel-on-appengine-standard/index.md]: @tjwebb , I'll merge this as soon as you make the recommended change.

**Comment by ToddKopriva** [comment]
@bshaffer , would you approve this PR, since you requested a change?

**Comment by bshaffer** [comment]
Done! Thanks everyone!

**Comment by ToddKopriva** [comment]
This fix will be published on cloud.google.com within a day or so.

**Pull request merged**